packages/init/src/steps/install-required-dependencies.ts

```diff
@@ -2,17 +2,33 @@ import path from "path";
 import { execCommand } from "../utils";
 import { Utils, FileSystem } from "@slicemachine/core";
 
-export async function installSm(cwd: string): Promise<void> {
+export async function installRequiredDependencies(
+  cwd: string,
+  framework: Utils.Framework.FrameworkEnum
+): Promise<void> {
   const yarnLock = Utils.Files.exists(FileSystem.YarnLockPath(cwd));
-  const command = yarnLock ? "yarn add -D" : "npm install --save-dev";
+  const installDevDependencyCommand = yarnLock
+    ? "yarn add -D"
+    : "npm install --save-dev";
+  const installDependencyCommand = yarnLock ? "yarn add" : "npm install";
 
   const spinner = Utils.spinner("Downloading Prismic Visual Builder");
   spinner.start();
 
   const { stderr } = await execCommand(
-    `${command} ${Utils.CONSTS.SM_PACKAGE_NAME}`
+    `${installDevDependencyCommand} ${Utils.CONSTS.SM_PACKAGE_NAME}`
   );
 
+  switch (framework) {
+    case Utils.Framework.FrameworkEnum.react:
+    case Utils.Framework.FrameworkEnum.next:
+      `${installDependencyCommand} ${Utils.CONSTS.PRISMIC_REACT_PACKAGE_NAME}`;
+      break;
+    case Utils.Framework.FrameworkEnum.svelte:
```

Comment:
I guess later on we could include `prismic/client`, but configuring it could be tricky.

Reply:
Also what about the slice-zone